### DISCUSSION TITLE
AddConsoleExporter() method not working in .NET6 WPF application

### DISCUSSION THREAD

**Discussion by SanyP23** [open-telemetry/opentelemetry-dotnet#4674] [Q&A]
Hi,

AddConsoleExporter() method not working in .NET6 WPF or .NET6 winforms application. It works upto .NET framework 4.7.2, but for .NET 6 it only generates log if it is a console app. Is it not supported for .NET 6 WPF application?

**Comment by utpilla** [comment]
@SanyP23 Could you update the output target for the ConsoleExporter to `Debug` and check the Debug window?

```csharp
.AddConsoleExporter(options => options.Targets = ConsoleExporterOutputTargets.Debug);
```
